### PR TITLE
Add more details to the docs

### PR DESCRIPTION
**Pull request by Kidswiss**
Add more points to the docs according to feedback I received.

### CODE REVIEW

File: docs/modules/ROOT/pages/object-storage/usage.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
+= How to Use an AppCat Object Bucket
```
Comment (akosma) [inline docs/modules/ROOT/pages/object-storage/usage.adoc:1]: I suggest we use K8up as an example here. Just the definition of a one-time `Backup` object referencing the secret parameters. What do you think?

File: docs/modules/ROOT/pages/object-storage/troubleshooting.adoc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+= Common Troubleshooting
```
Comment (akosma) [inline docs/modules/ROOT/pages/object-storage/troubleshooting.adoc:1]: I suggest we separate this page into another PR, until the workflow is clearly defined.

File: docs/modules/ROOT/pages/object-storage/secrets.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+[source,yaml]
+----
+$ oc get secrets objectbucket-creds -oyaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: objectbucket-creds
+data:
+  AWS_ACCESS_KEY_ID: OFk4Qzc0TzFHME9PNjkyWkFGRjU= <1>
+  AWS_REGION: cm1h <2>
```
Comment (akosma) [inline docs/modules/ROOT/pages/object-storage/secrets.adoc:14]: What is this region? Maybe better to use actual values from cloudscale.ch?

File: docs/modules/ROOT/nav.adoc [hunk 4]
```diff
@@ -1,4 +1,11 @@
 include::user:ROOT:partial$shared-nav.adoc[]
 
+.AppCat
+* xref:appcat/availability.adoc[]
+
 .Object Storage Buckets
 * xref:object-storage/how-to.adoc[]
```
Comment (tobru) [inline docs/modules/ROOT/nav.adoc:7]: Maybe rename to `create.adoc`?

File: docs/modules/ROOT/pages/appcat/availability.adoc [hunk 5]
```diff
@@ -0,0 +1,19 @@
+= Zone Availability of Services
+
+.Table Availability
```
Comment (tobru) [inline docs/modules/ROOT/pages/appcat/availability.adoc:3]:
```suggestion
```

Please add some table styling and table semantics (`cols`, `options=header`, ...) and I'd leave the title away

File: docs/modules/ROOT/pages/object-storage/delete.adoc [hunk 6]
```diff
@@ -0,0 +1,8 @@
+= Delete an Object Storage Bucket
+
+[source,bash]
+----
+$ oc delete objectbucket my-bucket
+----
+
+IMPORTANT: If you delete the objectBucket object from the cluster, it will delete the actual bucket as well!
```
Comment (tobru) [inline docs/modules/ROOT/pages/object-storage/delete.adoc:8]:
```suggestion
IMPORTANT: If you delete the `ObjectBucket` object from the cluster, it will delete the actual bucket as well!
```

Maybe also document the deletion policy on this page?

File: docs/modules/ROOT/pages/object-storage/secrets.adoc (hunk 3 above)
Comment (tobru) [reply docs/modules/ROOT/pages/object-storage/secrets.adoc:14]: This looks like base64 encoded? That's how secrets work...

https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl/#decoding-secret

File: docs/modules/ROOT/pages/appcat/availability.adoc (hunk 5 above)
Comment (tobru) [reply docs/modules/ROOT/pages/appcat/availability.adoc:3]: Oh, I see, it's just a draft... Never mind

File: docs/modules/ROOT/pages/object-storage/secrets.adoc (hunk 3 above)
Comment (akosma) [reply docs/modules/ROOT/pages/object-storage/secrets.adoc:14]: Ahhhh  😆  I thought it was the code of a region… then it's all good

Comment (Kidswiss) [reply docs/modules/ROOT/pages/object-storage/secrets.adoc:14]: Should we add a note or something about the base64 encoding?

File: docs/modules/ROOT/nav.adoc (hunk 4 above)
Comment (Kidswiss) [reply docs/modules/ROOT/nav.adoc:7]: I don't see the reasoning behind this rename? The page is about what's available where and not about creating stuff?

File: docs/modules/ROOT/pages/object-storage/troubleshooting.adoc (hunk 2 above)
Comment (Kidswiss) [reply docs/modules/ROOT/pages/object-storage/troubleshooting.adoc:1]: I've removed the file from this PR.

File: docs/modules/ROOT/pages/object-storage/usage.adoc (hunk 1 above)
Comment (Kidswiss) [reply docs/modules/ROOT/pages/object-storage/usage.adoc:1]: I've added a k8up backup as an example. I feel it's not quite optimal though...

What do you think?

File: docs/modules/ROOT/pages/object-storage/delete.adoc (hunk 6 above)
Comment (Kidswiss) [reply docs/modules/ROOT/pages/object-storage/delete.adoc:8]: I've added a note that it will be available later.

File: docs/modules/ROOT/nav.adoc (hunk 4 above)
Comment (tobru) [reply docs/modules/ROOT/nav.adoc:7]: The pages' title is "Create an Object Storage Bucket" and it would make sense to not have everything on one page, when now having multiple pages for "jobs to be done".

File: docs/modules/ROOT/pages/object-storage/usage.adoc (hunk 1 above)
Comment (akosma) [reply docs/modules/ROOT/pages/object-storage/usage.adoc:1]: Precisely! This is perfect. Thanks!

File: docs/modules/ROOT/pages/object-storage/create.adoc [hunk 7]
```diff
@@ -45,6 +45,8 @@ If you see the "True" value in the READY column, it means that your bucket is pr
 +
 IMPORTANT: If your object bucket doesn't get the state ready after a while, try to delete the bucket and re-create it with a different name.
 
+PROBABLY JUST LINK TO THE SECRETS PAGE HERE?
```
Comment (akosma) [inline docs/modules/ROOT/pages/object-storage/create.adoc:48]: Please mark these texts with the "TODO:" prefix in the future, to make them easier to spot in the text

**Comment by akosma** [comment]
LGTM! Ready to merge

File: docs/modules/ROOT/pages/object-storage/create.adoc (hunk 7 above)
Comment (Kidswiss) [reply docs/modules/ROOT/pages/object-storage/create.adoc]: Will keep in mind, thanks! 👍

**Comment by Kidswiss** [comment]
I'll merge when tobru gives his approval, too.

**Comment by tobru** [comment]
I did some cleanup and fixing in 0af1edfcfeb42145a10ad4bfd8b44edb38bc0546, please check. If you're OK with it, feel free to merge.

**Review by tobru — APPROVED** [review]

File: docs/modules/ROOT/pages/object-storage/create.adoc [hunk 8]
```diff
@@ -2,9 +2,7 @@
 
 AppCat allows DevOps engineers to quickly provision S3-compatible Object Storage Buckets in various environments.
 
-IMPORTANT: In its first release, AppCat Object Storage buckets are only available on the https://portal.appuio.cloud/zones/cloudscale-lpg-2[cloudscale.ch LPG 2 region] of APPUiO Cloud.
```
Comment (Kidswiss) [inline docs/modules/ROOT/pages/object-storage/create.adoc:5]: This statement is currently still true... Until we finish up the billing... So probably another sprint.

Comment (tobru) [reply docs/modules/ROOT/pages/object-storage/create.adoc:5]: Yes, but I feel it's on the wrong page, now that we have an availability overview in index.adoc...

Comment (Kidswiss) [reply docs/modules/ROOT/pages/object-storage/create.adoc:5]: Should we remove it from the availability page then, until it's really there?

Comment (tobru) [reply docs/modules/ROOT/pages/object-storage/create.adoc:5]: Done in b78a59a08222ebfbfbd723c4d8c1f3a0615db555
